org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/xpect/nonBooleanValueInIfCondition.wlk.xt

```diff
@@ -42,7 +42,7 @@ object p {
 
 		var a = 2
 		const b = 2
-		// XPECT errors --> "Expecting a boolean" at "a = b"
+		// XPECT errors --> "You did an assignment inside an if condition. Maybe you wanted to use '==' instead of '=' ?" at "a = b"
```

Comment:
Está bueno el mensaje indicando cómo corregir, pero le faltaría indicar bien que "no anda" si no lo cambiás.
```suggestion
		// XPECT errors --> "You did an assignment inside an if condition. This won't work, a boolean expression is expected. Maybe you wanted to use '==' instead of '=' ?" at "a = b"
```